policy/modules/system/systemd.te

```diff
@@ -1417,3 +1417,516 @@ optional_policy(`
+files_create_var_run_dirs(systemd_homed_t)
+# AVC avc:  denied  { create } for  pid=1000 comm="systemd-homewor" name="L_7:0" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:var_run_t:s0 tclass=file permissive=0
+# If I run test script it shows:
+# Would relabel /run/cryptsetup/L_7:0 from system_u:object_r:devicekit_var_run_t:s0 to system_u:object_r:var_run_t:s0
+# so whatever creates the file uses devicekit_var_run_t as context, so make systemd-homework to use it too.
+optional_policy(`
+	gen_require(`
+		type devicekit_var_run_t;
+	')
+	filetrans_pattern(systemd_homed_t, var_run_t, devicekit_var_run_t, file)
+	manage_files_pattern(systemd_homed_t, devicekit_var_run_t, devicekit_var_run_t)
+')
+
+
+
+#
+# netlink
+#
+
+#### Allow: systemd-homed to create and use netlink
+# AVC avc:  denied  { create } for  pid=1025 comm="systemd-homed" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=netlink_kobject_uevent_socket permissive=1
+# AVC avc:  denied  { setopt } for  pid=1025 comm="systemd-homed" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=netlink_kobject_uevent_socket permissive=1
+# AVC avc:  denied  { bind } for  pid=1025 comm="systemd-homed" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=netlink_kobject_uevent_socket permissive=1
+# AVC avc:  denied  { getattr } for  pid=1025 comm="systemd-homed" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=netlink_kobject_uevent_socket permissive=1
+# AVC avc:  denied  { read } for  pid=54703 comm="systemd-homed" scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=netlink_kobject_uevent_socket permissive=1
+allow systemd_homed_t self:netlink_kobject_uevent_socket create_socket_perms;
+
+
+
+#
+# dbus
+#
+
+# AVC avc:  denied  { acquire_svc } for  scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:system_r:system_dbusd_t:s0-s0:c0.c1023 tclass=dbus permissive=1 "
+optional_policy(`
+	dbus_acquire_svc_system_dbusd(systemd_homed_t)
+')
+
+# AVC avc:  denied  { send_msg } for  scontext=system_u:system_r:systemd_sleep_t:s0 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=dbus permissive=0
+allow systemd_sleep_t systemd_homed_t:dbus send_msg;
+
+#### Allow: Reboot/login
+# AVC avc:  denied  { send_msg } for  scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=dbus permissive=0
+# AVC avc:  denied  { connectto } for  pid=1000 comm="sshd" path="/run/systemd/userdb/io.systemd.Home" scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=unix_stream_socket permissive=0
```

Comment:
This should probably be addressed on a higher level. I forgot where I addressed it exactly. (maybe pam  or nss, probably pam)